TACC/Core-Portal
# task/FP-1130: Toggle Search

## Overview: ##
[A2CPS] Toggle Projects Search

## Related Jira tickets: ##

* [FP-1130](https://jira.tacc.utexas.edu/browse/FP-1130)

## Summary of Changes: ##

- Hide search bar in Shared Workspaces, if either a setting is declared
- Disable indexing of any Shared Workspaces/Projects, via a setting 

## Testing Steps: ##
1. Set **_PORTAL_DATAFILES_STORAGE_SYSTEMS** **hideSearchBar** property to **True** in settings_default.py
2. Navigate to Shared Workspaces, Search bar should not be rendered

## UI Photos:

<img width="1282" alt="Screen Shot 2021-08-20 at 3 11 26 PM" src="https://user-images.githubusercontent.com/62672123/130288450-67f28ebc-f2a2-4deb-aed5-cc40c346473b.png">


## Notes: ##
